backend/src/main/java/com/example/subjecthub/repository/SubjectCommentRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.example.subjecthub.repository;
+
+import com.example.subjecthub.entity.Subject;
+import com.example.subjecthub.entity.SubjectComment;
+import org.springframework.data.repository.CrudRepository;
+import java.util.List;
+
+public interface SubjectCommentRepository extends CrudRepository<SubjectComment, Long> {
+
+    List<SubjectComment> findBySubject_Id(Long subject_id);
```

Comment:
Even though this is on SQL territory, the method parameters should still use camelCase. Spring Data JPA uses underscores to traverse relationships which is the only reason why we have them.

Reply:
fixed